pillar/nginx/ocw_origin.sls

```diff
@@ -65,4 +65,6 @@ nginx:
                         - $uri/
                         - index.htm
                         - index.html
-                        - =404
```

Comment:
We don't want to remove this line as that is what will return the 404 if the file isn't found in the other locations.